views/userAllPackage.ejs

```diff
@@ -0,0 +1,44 @@
+<div class="container" id="search">
+<% if(packages.length >0){%>
+<form class="form my-3" action="#" method="GET">
+	<div class="input-group">
+    <input class="form-control form-control-lg" type="text" placeholder="Search Swift Package Registry" id="term" name="term">
+    <span class="input-group-append">
+    	<button type="submit" class="btn btn-primary" disabled>Search</button>
+    	</span>
+    </input>
+    </div>
+</form>
+<h2>Swift Packages</h2>
```

Comment:
Set title to "`<Username>`'s Swift Packages"

Reply:
Done this one.